backend/src/bundles/users/user.entity.ts

```diff
@@ -65,10 +65,12 @@ class UserEntity implements Entity {
     public toObject(): {
         id: number;
         email: string;
+        passwordHash: string;
```

Comment:
Remove passwordHash.
Depending on the return statement https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/blob/d7721ebfa699a87a1ac4d8578dc1c5053b991efe/backend/src/bundles/users/user.service.ts#L42C14-L42C14, we can return passwords to the client in the browser. I think this is a bad idea.

Reply:
I use findByEmail method inside auth-service to validate password. 
If I will remove it from return value I coludn`t use it.
We can switch usage of user-service into user-repository inside auth-service to fix that, but I am not sure about that

Reply:
Oh, to compare the password, we need return it. Sorry, my mistake.